client/me/security-u2f-key/add.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import u2f from './u2f-api';
+import Spinner from 'components/spinner';
+
+class SecurityU2fKeyAdd extends React.Component {
+	static propTypes = {
+		onRegister: PropTypes.func.isRequired,
+		onCancel: PropTypes.func.isRequired,
+		registerRequests: PropTypes.object.isRequired,
+	};
+
+	componentDidMount = () => {
+		this.registerKey();
+	};
+
+	registerKey = () => {
+		// const appId = this.props.registerRequests.appId;
+		delete this.props.registerRequests.appId;
+		// u2f.register( appId, [ this.props.registerRequests ], [], this.keyRegistered );
+		u2f.register(
+			'https://calypso.localhost:3000',
+			[ this.props.registerRequests ],
+			[],
+			this.keyRegistered
+		);
+	};
+
+	keyRegistered = data => {
+		this.props.onRegister( data );
+	};
+
+	createChallenge() {
```

Comment:
This method appears to be unused. Is that correct?

If it is used, it looks like a good candidate to prefer [`crypto`](https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto).

Reply:
removed